WalletWasabi/Crypto/GeExtensions.cs

```diff
@@ -0,0 +1,25 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace NBitcoin.Secp256k1
+{
+	public static class GeExtensions
+	{
+		public static bool IsGenerator(this GE me)
```

Comment:
as mentioned in slack, we will need multiple generator points:

1. two for Pedersen commitments `G_g` and `G_h`. Perhaps it makes sense to use the same points as [libsecp256k1-zkp's Pedersen commitments](https://github.com/ElementsProject/secp256k1-zkp/blob/1a1a803b7f06032aff6aba5d1de8f574b61e2fcd/src/modules/rangeproof/main_impl.h#L87-L114), in which case `G_h` would be the standard generator used for ECDSA (referred to as `G` in the libsecp256k1-zkp source) and `G_g` (referred to as `G2`) is derived by [hashing the standard generator](https://github.com/ElementsProject/secp256k1-zkp/blob/1a1a803b7f06032aff6aba5d1de8f574b61e2fcd/src/modules/rangeproof/main_impl.h#L16-L32). The reason the standard ECDSA generator is used for *blinding* is because in confidential transactions a balance of 0 is proven by providing an ECDSA signature with the sum of the commitments as the public key, `G_g` gets canceled out because the value is 0, and the sum of the randomness is the private key. We can't use a signature as a proof in the same way because of the additional randomization of commitments during the `Show` protocol.
2. 7 additional generators: `G_w`, `G_w'`, `G_x0`, `G_x1` and `G_V` used for the MAC/issuer parameters, `G_a` used for blinding the attribute, and `G_s` used for the serial number proof.

Reply:
Please take a look at this to better understanding of what @nothingmuch is talking about: https://github.com/lontivero/Wabisabi/blob/9cd24f8c175acf97c3f9c49b077a8714af178745/Wabisabi/Generators.cs#L13-L28